wdl-ast/CHANGELOG.md

```diff
@@ -7,8 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 0.15.1 - 07-31-2025
+
+#### Added
+
+* Added support used by LSP hovers and completions ([#519](https://github.com/stjude-rust-labs/wdl/pull/519)).
+
 ## 0.15.0 - 07-30-2025
 
+TODO: yank this?
+
```

Comment:
Stray TODO

Reply:
And no, I don't think we should yank it unless using it would somehow be unsafe.